controllers/produtocontroller.php

```diff
@@ -0,0 +1,102 @@
+<?php
+namespace Controllers;
+use Lib\Controller;
+use Models\Produto;
+use Lib\Session;
+use Lib\App;
+use Lib\Router;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+/**
+ * Description of ProdutoController
+ *
+ * @author Ga_Ca
+ */
+class ProdutoController extends Controller {
+      public function index(){
+          $this->data['produtos']= Produto::getProdutos(false);
+          
+      }
+      public function ver($idProduto){
+          $idProduto= filter_var($idProduto,FILTER_SANITIZE_NUMBER_INT);
+          if($idProduto!=FALSE){
+              $this->data['produto']= Produto::getProdutosPorId($idProduto);
+          }
```

Comment:
se `$idProduto` for `false` a view será carregada sem o índice `produto`e irá causar erro na view.